.nvmrc

```diff
@@ -1 +1 @@
-14.17.0
+18.15.0
```

Comment:
As we are using asdf `.tool-versions`, do we need to keep this? 🤔

Reply:
Good catch, removed all unused files in https://github.com/nimblehq/nimble-survey-web/pull/84/commits/37ee301bd357e6629e41ce48dda596bdd13b4f93